cmd/ptpcheck/cmd/sources.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/facebook/time/cmd/ptpcheck/checker"
+	"github.com/facebook/time/phc"
+	ptp "github.com/facebook/time/ptp/protocol"
+
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	sourcesNoDNSFlag bool
```

Comment:
Do you insist on having "Flag" in the variable name? I know it's in many places in ptpcheck but it's never too late ;)